v2.1/import.md

```diff
@@ -256,12 +323,27 @@ WITH
 		name TEXT,
 		INDEX name_idx (name)
 )
-CSV DATA ('azure://acme-co/customer-import-data.tsc?AZURE_ACCOUNT_KEY=hash&AZURE_ACCOUNT_NAME=acme-co')
+CSV DATA ('azure://acme-co/customer-import-data.csv?AZURE_ACCOUNT_KEY=hash&AZURE_ACCOUNT_NAME=acme-co')
 WITH
 	nullif = ''
 ;
 ~~~
 
+### Import a compressed CSV file
+
+{% include copy-clipboard.html %}
+~~~ sql
+> IMPORT TABLE customers (
+		id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+		name TEXT,
+		INDEX name_idx (name)
+)
+CSV DATA ('azure://acme-co/customer-import-data.gz?AZURE_ACCOUNT_KEY=hash&AZURE_ACCOUNT_NAME=acme-co')
```

Comment:
it would probably be named `customer-import-data.csv.gz`
That said, if the file ends in `.gz` you don't actually need the decompress option at all since we could figure it out form the filename, so I'm not sure what makes sense as an example.

Reply:
But we do have options `bzip` and `gzip` when they want to explicitly specify the codec, right?

Reply:
sure, you can pass `with decompress = 'gzip'` but you don't _need_ to if the filename is `.gz` since it would pick that anyway, so as an example, i'm not sure how much it communicates.

Reply:
Hmmm..so what would be a better example?

Reply:
I'm not sure... maybe no extension?

Reply:
we could also call the example "force selection of gzip decompression codec" or something to make it clearer why you'd pass the option

Reply:
Now I am wondering why we added the options in the first place 🤔

Reply:
the option is there so that if your filename _doesn't_ end in the common extension, you can force it to use a specific codec, but the default is to try to just figure it out.

Reply:
Hmm..so the default way we want users to import decompressed files is by not specifying any options. I am going to reframe the example to reflect that.